specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -589,6 +1116,112 @@ components:
         valuta:
           $ref: "#/components/schemas/Waardelijst"
           description: "Waardelijst wordt gepubliceerd op http://www.kadaster.nl/schemas/waardelijsten/Valuta"
+    Beperkingsgebied:
+      type: "object"
+      properties:
+        identificatie:
+          type: "string"
+        werkingsgebied:
+          type: "object"
+          description: "Datgene waarop de beperking van toepassing is."
+          properties:
+            bgtObjectIdentificatie:
+              type: "string"
+            bagObjectIdentificatie:
+              type: "string"
+            kadastraalObjectIdentificatie:
+              type: "string"
+            vrijeContour:
+              $ref: "http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/polygonGeoJSON.yaml"
+    Beslag:
+      type: "object"
+      description: "Tijdelijke maatregel opgelegd door een partij waardoor de zakelijk gerechtigde niets met de kadastraal onroerende zaak kan doen.
+
+
+        Waardelijst in deze component :
+          [omschrijvingBetrokkenRecht](http://www.kadaster.nl/schemas/waardelijsten/BetrokkenRecht/) en [aard](http://www.kadaster.nl/schemas/waardelijsten/AardBeslag/)"
+      properties:
+        identificatie:
```

Comment:
Zijn identificate & domein nodig?
We hebben nu namelijk geen apart endpoint om een beslag los te bevragen.
Of moet dit endpoint nog toegevoegd worden? (/kadastraalonroerendezaken/{koi}/beslagen/{beslag_identificatie})